server/controllers/documents.js

```diff
@@ -2,17 +2,15 @@ import db from '../models';
 import Paginator from '../helpers/pagination';
 
 /**
- *
- * @param {object} req
- * @param {object} res
- * @returns{object} response object
+ * Class for handling document operations
  */
 class DocumentController {
 
   /**
+   * Create Document
    *
-   * @param {object} req
-   * @param {object} res
+   * @param {object} req being sent
```

Comment:
whats being sent??? you can find another way to explain this I think e.g. `request being sent`